docs/modules/ROOT/pages/how-tos/logging/increase-elasticsearch-storage-size.adoc

```diff
@@ -0,0 +1,283 @@
+= Increase Storage Size of an OCP Logging Elasticsearch instance
+
+[abstract]
+This page describes how to increase the underlying storage size of the OpenShift Cluster Logging Elasticsearch instance.
+
+== Starting situation
+
+* You already have an OpenShift 4 with Cluster Logging enabled.
+* The Cluster Logging instance is managed and is of type `elasticsearch`.
+* You have admin-level access to the cluster.
+* You want to increase the storage size of the Elasticsearch cluster.
+
+== Prerequisites
+
+* `kubectl`
+* `curl`
+* `jq`
+* `yq` https://mikefarah.gitbook.io/yq[yq YAML processor] (version 4 or higher)
+* `commodore`, see https://syn.tools/commodore/running-commodore.html[Running Commodore]
+
+== Prepare local environment
+
+. Configure API access
++
+[source,bash]
+----
+export COMMODORE_API_URL=https://api.syn.vshn.net <1>
+export COMMODORE_API_TOKEN=<your lieutenant token>
+
+# Set Project Syn cluster and tenant ID
+export CLUSTER_ID=<lieutenant-cluster-id> # Looks like: c-cluster-id-1234
+export TENANT_ID=$(curl -sH "Authorization: Bearer ${COMMODORE_API_TOKEN}" ${COMMODORE_API_URL}/clusters/${CLUSTER_ID} | jq -r .tenant)
+----
+<1> Replace with the API URL of the desired Lieutenant instance.
+
+. Create a local directory to work in and compile the cluster catalog
++
+[source,bash]
+----
+export WORK_DIR=/path/to/work/dir
+mkdir -p "${WORK_DIR}"
+pushd "${WORK_DIR}"
+
+commodore catalog compile "${CLUSTER_ID}"
+----
++
+[TIP]
+====
+We strongly recommend creating an empty directory, unless you already have a work directory for the cluster you're about to work on.
+This guide will run Commodore in the directory created in this step.
+====
```

Comment:
I think here we can just link to https://syn.tools/syn/how-tos/compile-catalog.html

Reply:
I'm not a fan. In my experience it's way easier to follow a guide on one page and not have to jump between 2-3 pages.

It would be a bit inconsistent too. All other mentions in the docs show how to compile commodore directly on the page and don't link to syn.tools.

Reply:
Ok, but at least make a partial, otherwise this is unmaintainable.

Reply:
👍